CODEOWNERS

```diff
@@ -0,0 +1 @@
+*.sol @pdyraga @nkuba @lukasz-zimnoch
```

Comment:
Let's extract `CODEOWNERS` change to a separate PR and merge it to `main`. 

The `CODEOWNERS` should cover all existing contracts from `contracts/` directory

Let's add there all developers who worked on the Solidity code of the Threshold smart contracts: @vzotova @cygnusv @pdyraga 

This will not only protect dApp-dev-only changes from being merged but will also protect from any accidental changes to an already audited code.

Reply:
https://github.com/threshold-network/solidity-contracts/pull/121